dll/keyboard/kbdrost/kbdrost.spec

```diff
@@ -0,0 +1 @@
+@ stdcall KbdLayerDescriptor()
```

Comment:
Add an empty newline here.

Reply:
But... this is just copy-pasted, and from what I can see, none of the other layouts have trailing empty lines in there.